npm/react/plugins/next/findNextWebpackConfig.js

```diff
@@ -3,6 +3,7 @@
 const debug = require('debug')('@cypress/react')
 const getNextJsBaseWebpackConfig = require('next/dist/build/webpack-config').default
 const { findPagesDir } = require('../../dist/next/findPagesDir')
+const { getRunWebpackSpan } = require('../../dist/next/getRunWebpackSpan')
```

Comment:
Can we avoid referencing the `dist` folder from the plugins?
My 2 reasons:
- `dist` does not always exist (sometimes it's called `out` sometimes build)
- It took me a good 10 min to fin out why I could not test your repo as I thought plugins were not transpiled.

If we choose to transpile the plugins. We should do it entirely.
I believe this is confusing.

Instead could you replace your ts code with js code.

Another piece of info you might need: @cowboy and the DX team plan on moving all those "plugins" in their own package as soon as they can. I would hold off refactoring everything too fast.

Reply:
I'm following the advice based on the review of [this PR](https://github.com/cypress-io/cypress/pull/18009#pullrequestreview-748333727). I personally think adding a small amount of javascript is fine with the context of fixing a small bug but I've already started writing typescript so I carried on that trend for this PR.

Reply:
I see, sorry about the confusion. 
I will check directly with her.

Reply:
This isn't different than the existing patterns we have. Yes, we will be redoing these servers but not in the next 3 months.